src/main.js

```diff
@@ -2,7 +2,24 @@ import 'bootstrap';
 import 'bootstrap/dist/css/bootstrap.min.css';
 import './styles.css';
 import $ from "jquery";
+import { NekoPicApiCall } from './nekopicApi.js';
 
-$(document).ready(function(){
 
+$(document).ready(function () {
+  
+  $(".start").click(function () {
+    NekoPicApiCall().then(function (response) {
+      console.log(response)
+      function randomNumber(max){
```

Comment:
please move this `randomNumber` function outside of the `click` part. It doesn't need to be here.